src/server.js

```diff
@@ -16,7 +15,7 @@ const {MongoDatabaseManager} = require('./utils/mongoDatabaseManager');
 async function createServer(fnCreateContainer) {
     await new MongoDatabaseManager().connectAsync();
 
-    const app = createApp({fnCreateContainer, trackMetrics: isTrue(env.TRACK_METRICS)});
+    const app = createApp({fnCreateContainer, trackMetrics: true});
```

Comment:
IMO: We should keep this environment variable and disable Prometheus till prod env after team discussion.

Reply:
Added back.